### PR TITLE
catalog install & up multiline env

### PR DESCRIPTION
**Pull request by amy**
https://github.com/rancher/rancher/issues/9305

### CODE REVIEW

File: lookup/fileenv.go [hunk 1]
```diff
@@ -82,23 +79,49 @@ func parseCustomEnvFile(file string, parent config.EnvironmentLookup) (*FileEnvL
...
+}
+
+func NewFileEnvLookup(file string, parent config.EnvironmentLookup) (*FileEnvLookup, error) {
+
+	v, err := ParseEnvFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	variables := map[string]string{}
```
Comment (joshwget) [inline lookup/fileenv.go:114]: Couldn't `ParseEnvFile` just return `map[string]string`?

Comment (amy) [reply lookup/fileenv.go:114]: ParseEnvFile is used elsewhere that requires type `map[string]interface{}`

**Comment by joshwget** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**